app/config/config.yml

```diff
@@ -112,7 +113,7 @@ nelmio_security:
     referrer_policy:
         enabled: true
         policies:
-          - no-referrer
+          - no-referrer-when-downgrade
```

Comment:
Allows it to work in Chrome 56-60 etc (don't think we need to be _quite_ so strict).

Reply:
Are we ever downgrading? Even `crm` has HTTPS, apart from ORCID

Reply:
Well, locally

Reply:
Clicking a `http://` link.

Reply:
I think it makes sense to loosen it just this much in order to get things working. Nothing should be able to leak outside of a TLS context, even with this setting.